### PR TITLE
Stubbed out api calls

### PR DESCRIPTION
**Pull request by ghmeier**
* moved `containers` to `models`
* updated functionality for roaster accounts
* added stub methods for customer accounts and billing subscriptions.

### CODE REVIEW

**Comment by ghmeier** [comment]
@jonnykry i'll let you review and merge this :)

File: handlers/roaster_account.go [hunk 1]
```diff
@@ -19,74 +20,77 @@ type RoasterAccountIfc interface {
...
-		fmt.Printf("%s",err.Error())
+		fmt.Printf("%s", err.Error())
 		return
 	}
 
-  // TODO:  Update parameters
-	err = c.sql.Modify(
-		"INSERT INTO roaster_account VALUE(?)",
-		uuid.New())
+	// TODO: create stripe account?
```
Comment (jonnykry) [inline handlers/roaster_account.go:40]: Planning on picking up at this point next time I work here!

**Review by jonnykry — APPROVED** [review]
Thanks for working on this, looks good to me!

File: handlers/roaster_account.go (hunk 1 above)
Comment (ghmeier) [reply handlers/roaster_account.go:40]: dope. i think figuring out what we do when creating/updating/deleting accounts will be the semi-complex part here.

Comment (ghmeier) [reply handlers/roaster_account.go:40]: https://github.com/stripe/stripe-go

**Pull request merged**